lib/OpenQA/Utils.pm

```diff
@@ -305,15 +306,25 @@ my %bugrefs = (
 );
 my %bugurls = reverse %bugrefs;
 
+sub bugref_regex {
+    my $regex = join('|', keys %bugrefs);
+    return qr/(($regex)#(\d+))(?![\w\d]+)/;
```

Comment:
I would rather not put `\d` into the forward lookup. I guess its fine function wise, but it feels superfluous to me. "Accept all digits and then there shouldn't be any word character or digit". Why also matching more than one of these characters?